docs/compression_algorithms/Pruning.md

```diff
@@ -118,3 +118,86 @@ This approach allows pruning the model taking into account layer-specific sensit
 ```
 
 > **NOTE:**  In all our pruning experiments we used SGD optimizer.
+
+#### Filter pruning statistics
+A model compression can be measured by two main metrics: filter pruning level and FLOPs pruning level. While 
+filter pruning level shows the ratio of removed filters to the total number of filters in the model, FLOPs pruning level 
+indicates how the removed filters affect the number of floating point operations required to run a model. 
+
+During the algorithm execution several compression statistics are available. See the example below.
+```
+Statistics by pruned layers:
++----------------------+------------------+--------------+---------------------+
+|     Layer's name     |  Weight's shape  | Mask's shape |   Filter pruning    |
+|                      |                  |              |        level        |
++======================+==================+==============+=====================+
+| ConvBlock[conv1]/NNC | [192, 32, 1, 1]  | [192]        | 0.500               |
+| FConv2d[conv]        |                  |              |                     |
++----------------------+------------------+--------------+---------------------+
+| ConvBlock[conv2]/NNC | [384, 64, 1, 1]  | [384]        | 0.500               |
+
```

Comment:
Looks like part of table is missed

Reply:
It was a bit unclear for me, I think closing line or indicator like `...` or `etc` is more clear than previous option. I think current version is clear, thanks!